src/haddock/modules/analysis/clustfcc/__init__.py

```diff
@@ -145,13 +145,12 @@ def _run(self):
                     clt_dic[cluster_id].append(pdb)
 
             # Rank the clusters
-            #  they are sorted by the top4 models in each cluster
-            top_n = 4
+            #  they are sorted by the topX (threshold) models in each cluster
             score_dic = {}
             for clt_id in clt_dic:
                 score_l = [p.score for p in clt_dic[clt_id]]
                 score_l.sort()
-                top4_score = sum(score_l[:top_n]) / float(top_n)
+                top4_score = sum(score_l[:threshold]) / float(threshold)
```

Comment:
Is there somewhere a check that the threshold should be >= 1?

Reply:
just added!

Reply:
Is defined by the for loop and the `range`